components/k8s-model-server/iap-client/REAMDE.md

```diff
@@ -0,0 +1,60 @@
+# Serving with IAP enabled
+
+## Setup
+
+Follow this [doc](https://github.com/kubeflow/kubeflow/blob/master/docs/gke/iap.md) to
+setup the cluster with IAP enabled. Save the client id as `IAP_CLIENT_ID`
+
+Deploy a [TF serving component](https://github.com/kubeflow/kubeflow/tree/master/components/k8s-model-server)
+with a mnist model.
+```commandline
+ks init my-model-server
+cd my-model-server
+ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow
+ks pkg install kubeflow/tf-serving
+ks env add cloud
+MODEL_COMPONENT=serveMnist
+MODEL_NAME=mnist
+MODEL_PATH=gs://kubeflow-ci-test-models/mnist
+MODEL_SERVER_IMAGE=gcr.io/$(gcloud config get-value project)/model-server:1.0
```

Comment:
Why are you overriding the Docker image supplied by the ksonnet components?

Reply:
removed